### PR TITLE
add repeatSlice

### PR DESCRIPTION
**Pull request by 9il**
Depends on #4652 (needs recent changes in `ndslice.slice`)

### CODE REVIEW

File: std/experimental/ndslice/selection.d [hunk 1]
```diff
@@ -1793,3 +1794,98 @@ struct IotaMap()
...
     }
 }
+
+/++
+Returns a slice, the elements of which are always the same.
+`RepeatSlice` stores only single value.
+
+Note:
+    Comparing with $(REF repeat, std,range)
+
```
Comment (wilzbach) [inline std/experimental/ndslice/selection.d]: I think `See_Also` is more commonly used

File: std/experimental/ndslice/selection.d [hunk 2]
```diff
@@ -1793,3 +1794,98 @@ struct IotaMap()
         return index;
     }
 }
+
+/++
+Returns a slice, the elements of which are always the same.
+`RepeatSlice` stores only single value.
```
Comment (wilzbach) [inline std/experimental/ndslice/selection.d]: grammar nitpicks:
1) stores only __a__ single value
2) Maybe sth. like "Returns a slice with identical elements"

File: std/experimental/ndslice/selection.d [hunk 3]
```diff
@@ -1793,3 +1794,98 @@ struct IotaMap()
...
+    }
+    else static if (is(T : Unqual!T) && is(Unqual!T : T))
+        private alias UT = Unqual!T;
+    else
+        private alias UT = T;
+    private UT _value;
+
+    ref T opIndex()
+    {
+        return value;
```
Comment (wilzbach) [inline std/experimental/ndslice/selection.d]: shouldn't this be `_value`?

Comment (9il) [inline std/experimental/ndslice/selection.d]: Yes, thanks)

File: std/experimental/ndslice/selection.d (hunk 2 above)
Comment (9il) [inline std/experimental/ndslice/selection.d]: thanks)

**Comment by 9il** [comment]
will open after ndslice.algorithm is merged

**Pull request closed without merging**